server/app/views/questiontypes/RadioButtonQuestionRenderer.java

```diff
@@ -80,6 +84,7 @@ private DivTag renderRadioOption(
             .withName(selectionPath)
             .withValue(String.valueOf(option.id()))
             .withCondChecked(checked)
+            .condAttr(applicantSelectedQuestion(questionName, questionType), "autofocus", "")
```

Comment:
Suggestion (non-blocking):  Returning the suggestion you made to me of using `Attr.AUTOFOCUS`.

Reply:
good catch! i updated it in all the places.